.github/workflows/go-darwin.yml

```diff
@@ -22,7 +22,7 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Make
-        run: make
+      - name: Test
```

Comment:
I think the divergence you are introducing here between the linux and the mac/windows builds needs to be documented, either as just a comment here and in the windows yaml, or something else. I know the reason from a slack chat, but we need a better record of that longer term and for others.

There is also the issue that if build tags for OS or os-specific files (implicit tags) are used, we will not be checking coverage of them or linting them.

Reply:
ok, you are right.
I think more and more we should probably just have a single CI file and be done with it.

What do you think?

Reply:
you know what, I can live with this not being perfect. as mentioned earlier: testing for osx and windows is already a luxury imo.
however, I can change it to `make test check-coverage`
or even `make ci`
then `all: ci lint`
what do you think?

Reply:
Does not need to be perfect, just needs a comment as to why it is different. Can be fixed later if it matters.

Reply:
as you've said elsewhere, linting on CI is important, so a `ci` target in the makefile that does not so linting may raise an eyebrow or two.

So I think just a comment in the mac/win files saying:
```
# lint-with-docker doesn't work on mac/win github actions. Just run test only.
```
or something.